src/ess/reduce/nexus/types.py

```diff
@@ -232,7 +232,7 @@ class CalibratedDetector(sciline.Scope[RunType, sc.DataArray], sc.DataArray):
     """Calibrated data from a detector."""
 
 
-class CalibratedBeamline(sciline.Scope[RunType, sc.DataArray], sc.DataArray):
+class CalibratedDetectorBeamline(sciline.Scope[RunType, sc.DataArray], sc.DataArray):
```

Comment:
We spoke the other day about not breaking downstream and notebooks anymore by renaming (unless really necessary). Is this necessary? Is the new name better?